beanstalk_worker/worker_dispatcher.py

```diff
@@ -56,3 +58,7 @@
 
     logger.debug("Deleting job: %s" % job_details)
     job.delete()
+
+    # remove handler from logging object
+    if 'dfh' in locals():
```

Comment:
If we set the debug log level, info messages will anyway be logged - is your question same or you want to ask something else ?